### PR TITLE
Add feature mutation

### PR DESCRIPTION
**Pull request by narongdejsrn**
Add example of how to use mutation with RANGE_ADD config

### CODE REVIEW

**Comment by lvarayut** [comment]
Excellent! You might consider to change the mutation name from `Create` to `Add`, just to be consistent with the `AddFeatureComponent`.

File: client/mutation/CreateFeatureMutation.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import Relay from 'react-relay';
+
+class CreateFeatureMutation extends Relay.Mutation {
```
Comment (lvarayut) [inline client/mutation/CreateFeatureMutation.js]: This file must be placed under the `client/components/Feature` as `AddFeatureMutation.js`.

**Comment by narongdejsrn** [comment]
Done!

**Comment by lvarayut** [comment]
Thanks!

**Pull request merged**